### PR TITLE
Adding batch timestamp to the public tx API

### PR DESCRIPTION
**Pull request by otherview**
### Why this change is needed

- Transactions Request should have batch timestamp along side with batch number 
- Update the folder and test name from obscuro to ten
- Update tenscan to display the timestamps

### What changes were made as part of this PR

Please provide a high level list of the changes made

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [ ] PR checks reviewed and performed

### CODE REVIEW

**Comment by Jennievon** [comment]
> @Jennievon the timestamp that's displayed doesn't have any treatment (it's a unix timestamp). I wonder if you have some wizard trick to display these nicely ?

sure, I'll push an update for it.

File: tools/tenscan/frontend/src/components/modules/dashboard/recent-transactions.tsx [hunk 1]
```diff
@@ -17,6 +17,11 @@ export function RecentTransactions({ transactions }: { transactions: any }) {
                 #{transaction?.BatchHeight}
               </p>
             </div>
+            <div className="ml-4 space-y-1">
+              <p className="text-sm font-medium leading-none">
+                {transaction?.BatchTimestamp}
```
Comment (BedrockSquirrel) [inline tools/tenscan/frontend/src/components/modules/dashboard/recent-transactions.tsx:22]: Would it work to use `{formatTimeAgo(batch?.timestamp)}` here, like @Jennievon did in `recent-batches.tsx`? To get the 'x mins ago' formatting.

**Review by BedrockSquirrel — APPROVED** [review]
LGTM but maybe Jennie can advise on the formatting comment.

**Review by Jennievon — APPROVED** [review]
LGTM!

**Pull request merged**